_episodes/08-kokkos-gpu.md

```diff
@@ -126,18 +132,16 @@ these aspects to some extent.
 >      thermo_style custom step time  temp press pe ke etotal density v_acn
 >      run		500
 >      ```
->      Let us name this modified input file as
->      *Input2*. Run *Input2* using the same identical Kokkos setting:
->      `-k on g 4 -sf kk -pk kokkos newton off neigh full comm device cuda/aware off`
->      and with identical number of GPU and MPI tasks as you did for the *Input1*.
->   4. Again, measure the performance of this run in units of `timesteps/s`.
->   5. Compare the performance between these two runs and comment on your observations.
+>      {: .source}
+>
+> 3. Rename the input file and run it using the same Kokkos setting as before, and the identical
+>    number of GPU and MPI tasks as previously. Measure the performance of this run in `timesteps/s`
+>    and compare the performance of these two runs and comment on your observations.
 >
 > > ## Solution
-> > I did this study in a Intel Haswell CPU node having 2x12 cores per
-> > node and two NVIDIA K80 GPUs (which is four visible devices per node). I have used
-> > 1 MPI tasks per GPU. This means that for four visible devices we have used four MPI
-> > tasks in total (per node).
+> >
+> > Taking an example from a HPC system with 2x12 cores per node and 2 GPUs (4 visible devices per
+> > node), using 1 MPI task per GPU, the following was observed.
 > >
 > > First, we ran with *Input1*. Second, we modified this input as mentioned above (to
 > > become *Input2*) and performance for both of these runs are measured in units of
```

Comment:
@prithwish-ichec See L153 `|Input 1 (all Kokkos enabled styles used)| 8.097 |        |`. Should there be a blank space there?

Reply:
Yes this is fine. We are comparing the performance loss for Input2 w.r.t Input1.